CONTRIBUTING.md

```diff
@@ -175,29 +175,22 @@ Visit https://www.npmjs.com/package/@instana/${package-name}/access and set publ
 
 To make a release, you first need to ensure that the released version will either be a semver minor or patch release so that automatic updates are working for our users. Following that, the process is simple:
 
-- Acquire an OTP token for 2fa.
-- Ensure you are logged in with npm in your shell (`npm whoami`)
-- Run `NPM_CONFIG_OTP={your token} lerna publish && lerna bootstrap`.
+- Go to <https://github.com/instana/nodejs/actions/workflows/release.yaml>
+- click on the "Run workflow" dropdown
+- you do not need to change any settings (dry run etc.) the default values are fine
+- click on the "Run workflow" button to trigger the action
 
-Lerna will determine if this is going to be a major, minor or patch version from the commit comments of all commits since the last release. It will also automatically update the CHANGELOG.md files in the root of the repository and also in all individual packages, based on those commits.
-
-If you want to review the changes that lerna is about to apply (for example the CHANGELOG files), you can execute `lerna version --no-git-tag-version --no-push` first.
-
-For each package release, we also publishing a new Lambda layer and a Fargate Docker image layer. This happens automatically via CI.
+The Github action will try to publish new versions for all packages from the most recent commit of the `main` branch. It will check if there is at least one successful CircleCI build for that commit hash for the main `build` workflow as well as for the `legacy-nodejs-versions` workflow. If that check fails, the Github action will abort. Otherwise it will go on to publish new releases to npm accordingly.
 
-#### Separate lerna version And lerna publish
-
-You might want to separate the version bumping and tagging from publishing to the npm registry. This is also possible. The separate lerna publish command (see below) is also helpful if the publish did not go through successfully.
+Lerna will determine if this is going to be a major, minor or patch version from the commit comments of all commits since the last release. It will also automatically update the CHANGELOG.md files in the root of the repository and also in all individual packages, based on those commits.
 
-- Run `lerna version`.
-- Lerna will determine if this is going to be a major, minor or patch version from the commit comments of all commits since the last release. It will also update all relevant changelog files.
-- Lerna will push the commit and the tag created by `lerna version` to GitHub. Check that this has happened. Also check that the version numbers in package-lock.json have been updated as well.
-- Acquire an OTP token for 2fa.
-- `NPM_CONFIG_OTP={your token} lerna publish from-package && lerna bootstrap`
```

Comment:
Discussion:Wondering if we would like to keep this. The GH action can still run into this error and then we need to manually fix the situation.

Exactly for this case

> That last command (`lerna publish from-package`) can also be run if the previous publish command went through for a subset of packages but not for others. Lerna will automatically figure out for which packages the latest version is not present in the registry and only publish those.

Reply:
Yeah, I was 50/50 on this when I deleted this. I'll revert this deletion and add some verbiage that this is can be used if an automated release goes south. Ideally, I would like to move to a setup where we never publish anything to npm from a local machine, so might want a Github action that executes this, but for now it's probably better to keep this section.

Reply:
Also: if the action fails, do we get a mail or similar?

Reply:
Good point. You can enable notifications for actions for your account at https://github.com/settings/notifications (in the section titled "Actions"). But enabling that might also get you notifications for other repos you contribute to. I'll add that to the docs.

I guess I better also add a Slack notification to the action. I'll take care of that today.